compiler/control/CompileMethod.cpp

```diff
@@ -378,7 +382,16 @@ compileMethodFromDetails(
 
          // not ready yet...
          //OMR::MethodMetaDataPOD *metaData = fe.createMethodMetaData(&compiler);
-
+         if (compiler.compileRelocatableCode())
+            { 
+            const char* methodName = compilee.externalName(&trMemory);
+            TR::Compiler->aotAdapter->createAndRegisterAOTMethodHeader(
+                        methodName,
+                        startPC,
```

Comment:
`startPC` is set below isn't it?

Reply:
Right, will be shipped in the next commit. Thank you!